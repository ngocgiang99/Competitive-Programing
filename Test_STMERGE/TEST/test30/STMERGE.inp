2
7 9
822022590 704983547 153760676 397601045 60628472 235034903 542107246 506582405 430584829 
636749416 423134558 160561869 423211408 175593283 187877462 561492065 528262839 695267134 
800030590 299302903 741270456 263762477 842959673 565001604 596210967 840354342 118422572 
495765310 570015640 289972093 631979326 606743448 208857666 351982791 878008210 292353351 
826987610 245290714 492898957 853687106 299516771 621032650 932822485 685771453 965353763 
881034220 66190671 601987884 182113696 699013590 424334624 663182918 442055966 535307868 
8184236 426024958 382073889 7216532 405570969 98021514 653332336 786297298 572527563 
7 1
344277931 
652538728 
226069043 
349922116 
912193525 
218767317 
280450296 
