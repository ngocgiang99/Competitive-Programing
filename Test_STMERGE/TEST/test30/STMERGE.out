8184236
280450296
